### ISSUE TITLE
website: responsive layout broken at 320px breakpoint

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#2235]
### I verified there's no existing issue for this bug.

- [X] There are no existing issues

### Current behavior

Looks like the website layout got broken and has introduced horizontal scrolling somewhere along the line. Possibly in some of the more complex layout examples like toggle button group.

<img width="423" alt="Screenshot 2023-12-28 at 12 45 30 PM" src="https://github.com/eBay/skin/assets/38065/16af824c-15cb-45a8-b588-1dc85e985c19">

### Skin version

17.0.0

### Steps to reproduce

```bash
1.
2.
3.
...
```

**Comment by ArtBlue** [comment]
We can verify this to be the case, but this may not be an issue. If I remember correctly, this may have been one of the drawbacks of the combination of having minimum button widths imposed and not being able to use container queries. I vaguely recall some compromises such as this, but we can verify.

**Comment by ianmcburnie** [comment]
> We can verify this to be the case, but this may not be an issue. If I remember correctly, this may have been one of the drawbacks of the combination of having minimum button widths imposed and not being able to use container queries. I vaguely recall some compromises such as this, but we can verify.

I am just meaning that our Skin website itself shouldn't break at 320px. If you are saying this breakage is unavoidable/intentional then perhaps these layouts should be moved to a different page to avoid breaking the main skin page at 320. Feels like we need to demonstrate some kind of responsive solution though.

**Comment by agliga** [comment]
I believe this should be fixed by the redesign
We do have some components that break at <320, but those should be tackled at the individual component level

**Comment by ArtBlue** [comment]
> I believe this should be fixed by the redesign We do have some components that break at <320, but those should be tackled at the individual component level

Agreed. I think we can close this for now and deal with the specific issue individually as we see them.
